spec/game_runner_spec.rb

```diff
@@ -60,7 +60,7 @@
       
       it "after the user takes a turn, the computer takes a turn, and the grid is re-rendered with an O in a random location" do
         def seed_random_generator_for_computer_turn
```

Comment:
I didn't notice this method in the middle of the it block before. It should probably move into a support file, or at least into the highest describe block.

Reply:
Right. ✔️